### PR TITLE
Add CONTRIBUTING.md file

### LINKED ISSUE #2: Add a CONTRIBUTING file

### PR DESCRIPTION
**Pull request by bobbyiliev**
Adding a `CONTRIBUTING.md` file and a helper `init.sh` script that users can run to generate the default file structure for new examples.

Closes #2 

materialize-connection-examples

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -1,6 +1,50 @@
...
 
-Thank you for your interest in Materialize demos! Contributions of many kinds
-are encouraged and most welcome.
+Thank you for your interest in Materialize connection examples!
+Contributions of many kinds are encouraged and most welcome.
+
+## Pull requests
+
+We welcome pull requests from everyone. By participating in this project, you
+agree to abide by the [code of conduct](CODE_OF_CONDUCT.md).
```
Comment (morsapaes) [inline CONTRIBUTING.md:9]: Do we have a code of conduct somewhere?

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -1,6 +1,50 @@
...
+
+Fork, then clone the repo:
+
+```
+git clone https://github.com/MaterializeInc/connection-examples.git
```
Comment (morsapaes) [inline CONTRIBUTING.md:14]:
```suggestion
git clone https://github.com/<username>/connection-examples.git
```

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -1,6 +1,50 @@
...
+
+Once you have added your code, commit your changes and push to your fork:
+
+```
+git add .
+git commit -m "Add my example"
+git push origin my-branch
+```
+
+Then submit a pull request through the GitHub website.
```
Comment (morsapaes) [inline CONTRIBUTING.md:46]:
```suggestion
That's it — you're ready to submit a pull request!
```

File: CONTRIBUTING.md [hunk 4]
```diff
@@ -1,6 +1,50 @@
...
+
+## Issues
```
Comment (morsapaes) [inline CONTRIBUTING.md:48]:
```suggestion
## Getting support
```

File: CONTRIBUTING.md [hunk 5]
```diff
@@ -1,6 +1,50 @@
...
 
 If you have questions, please [create a GitHub issue](https://github.com/MaterializeInc/connection-examples/issues/new).
```
Comment (morsapaes) [inline CONTRIBUTING.md:50]:
```suggestion
If you run into a snag or need support as you prepare your contribution, join the Materialize [Slack community](https://materialize.com/s/chat) or [open an issue](https://github.com/MaterializeInc/connection-examples/issues/new)!
```

Comment (morsapaes) [reply CONTRIBUTING.md:50]: For consistency with [`/demos`](https://github.com/MaterializeInc/demos#getting-support).

**Review by morsapaes — APPROVED** [review]
The init script is such a great idea; thank you, @bobbyiliev!

File: CONTRIBUTING.md (hunk 1 above)
Comment (bobbyiliev) [reply CONTRIBUTING.md]: Ah yes, good catch. I've just removed this reference!

**Pull request merged**